include/LightGBM/c_api.h

```diff
@@ -180,8 +189,8 @@ DllExport int LGBM_DatasetSetField(DatesetHandle handle,
 * \param field_name field name
 * \param out_len used to set result length
 * \param out_ptr pointer to the result
-* \param out_type  float_32:0, int32_t:1
-* \return 0 when success, -1 when failure happens
+* \param out_type  float32 or int32
```

Comment:
The label is always float_32 even in cli version. The float/double data and label actually not affect much on accuracy.